Quickstart uninformative links and Grammarly edits (#189)
hubverse-org/hubDocs: docs/source/quickstart-hub-admin/using-hub.md

## Target data  

> Many hubs will focus on modeling tasks where the goal is to estimate or predict a quantity that is, in principle, observable. We refer to this as 'target data'. Click on the link to read [a detailed description of target data](../user-guide/target-data.md).  

Reviewer: ```suggestion
Many hubs will focus on modeling tasks where the goal is to estimate or predict a quantity that is, in principle, observable. We refer to this as 'target data', which is detailed in the [Target Data chapter](../user-guide/target-data.md).  
```
